wiki/Ranking_Criteria/osu!taiko/en.md

```diff
@@ -3,46 +3,161 @@ osu!taiko
+- **Big Kat / Big Blue Note / Finisher Note:**  A strong beat that can be hit with both Drum Rim keys at the same time for extra score.
+- **BPM:** An acronym for “beats per minute” used to determine the tempo of a song.
+- **Slider / Drumroll:** A yellow bar with ticks inside of it which are played with any of the assigned keys. These ticks are usually snapped to 1/4 corresponding to the song's BPM. If the BPM is lower than 125 this changes to 1/8 snapping and 1/2 snapping if the BPM is higher than 250. If slider tick rate is set to 3, the slider ticks will instead snap to 1/3.
+- **Spinner:** A round element that requires the player to alternate between hitting Don/Kat for a determined number of times, which is shown in the center of the spinner. This number depends on the Overall Difficulty of the map as well as the length of the spinner.
+- **Bar Line:** A line that appears on the playfield and determines the start of a measure if the BPM is higher than 75. If the BPM is smaller or equal to 75 bar lines will be drawn on every beat.
+- **Overlap:** A note that partially or totally covers other note(s) in the playfield.
+- **Rest Moment:** A period of time without notes used specifically to allow the player to rest their hands and prepare for the upcoming patterns.
+- **Stream:** Consecutive circles grouped together. Usually snapped to ¼ beats.
+- **Snapping:** Timeline tick where an object is placed.
+- **Variable Snapping:** A combination of multiple different ways to snap notes within a short span of time due to the song’s fluctuating nature at that point.
+- **Slider Velocity:** Speed at which the notes/drumrolls/spinners/bar lines move horizontally from right to left through the playfield. Base slider velocity can be controlled in the timing panel and additional changes can be made through inherited (green) timing points.
+- **Smooth Slider Velocity Changes:** A mechanic that enables gradual transition from lower to higher / from higher to lower Slider Velocity sections. Transition notes with varying Slider Velocities are used to achieve this effect.
+- **Improvisation:** Having more notes than the actual song is providing.
+- **Taiko Template Background:** Background images that simulate authentic Taiko no Tatsujin gameplay. They usually include a black bar showing artist and song title in white beneath the playfield.
 
-Rules
--------
+## General
 
-All rules are exactly that: **rules**. They are **not** guidelines and may **not** be broken under **any** circumstance.
+### Rules
 
--   **Finish notes must not be in the middle or at the beginning of a 1/4 or above snapped pattern.** This is to ensure playability of all patterns, as finisher notes in the middle or at the beginning of a 1/4 or above snapped pattern have been deemed as very issuesome to catch.
+All rules are exactly that: **rules**. They are **not** guidelines and must **not** be broken under **any** circumstance.
```

Comment:
circumstances rather than circumstance ?

Reply:
both are equally as clear in my opinion, no need to be plural as this enforces the rule under ONE circumstance while using plural would maybe send the message that they have to meet two or more circumstances in order to break a rule which is not true